scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/TypeExtensions.kt

```diff
@@ -10,14 +10,15 @@ the Change License after the Change Date as each is defined in accordance with t
 package io.seldon.dataflow
 
 import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.ExperimentalCoroutinesApi
```

Comment:
question: curious what is this adding?

Reply:
Some parallel operations on Flows (i.e parallel processing of elements in a flow, like the one we want to do for pipelines) are marked as experimental, with semantics that might change. We have a test that uses some of those to see how they behave.